src/MAUI/Maui.Samples/Samples/Layers/AddClusteringFeatureReductionToAPointFeatureLayer/AddClusteringFeatureReductionToAPointFeatureLayer.xaml.cs

```diff
@@ -137,14 +146,18 @@ private void DisplayLabelsCheckBox_CheckedChanged(object sender, CheckedChangedE
             {
                 _clusteringFeatureReduction.LabelDefinitions.Clear();
             }
-            
         }
 
-        private void UpdateClusteringProperties(object sender, EventArgs e)
+        private void ClusterRadiusPicker_SelectedIndexChanged(object sender, EventArgs e)
```

Comment:
Do we need two event handlers here? 

Previously calling `UpdateClusteringProperties` covered both bases.

Also I think we should maintain our existing format of:

```
if(_layer != null)
{
    // do something
}
```

I don't think bracketless if statements are used in this format anywhere else.

Reply:
I kept running into null reference exceptions (specifically WPF) when I modified `UpdateClusteringProperties` to account for the swapping of the sliders with pickers. The issue is `MaxScalePicker.SelectedItem` is null when setting `ClusterRadiusPicker.SelectedItem`. Technically we might be able to null check on the picker selected items before trying to use them, however that introduces extra logic that can be avoided by having two event handlers. Furthermore, using two events allows each control to handle their own property setting separately, avoiding unnecessary property assignments. Happy to use curly braces for the if statements.